lib/modules/auth/index.js

```diff
@@ -322,6 +322,23 @@ export default class Auth extends ModuleBase {
     return this._native.fetchProvidersForEmail(email);
   }
 
+  /**
+   * Sets the language for the auth module using the device language
+   * @returns {*}
+   */
+  useDeviceLanguage() {
+    return this._native.useDeviceLanguage();
```

Comment:
No need to return here or on languageCode

Reply:
Ah yes!